### PR TITLE
Add toggle to remove 68% legend labels

### PR DESCRIPTION
**Pull request by Zaharid**
When making a paper plot that needs to fit in half a column, legend
space is at a premium. Add a toggle to remove the "(68%)" giving the
option to clean up space when the legend wouldn't fit otherwise.

### CODE REVIEW

**Comment by siranipour** [comment]
It would be nice if you could do this for lumi plots too or is that a lot of work

File: validphys2/src/validphys/pdfplots.py [hunk 1]
```diff
@@ -438,11 +446,20 @@ def draw(self, pdf, grid, flstate):
             errorstdup, errorstddown = stats.errorbarstd()
             ax.plot(xgrid, errorstdup, linestyle='--', color=color)
             ax.plot(xgrid, errorstddown, linestyle='--', color=color)
-            label  = rf"{pdf.label} ($68%$ c.l.+$1\sigma$)"
+            label = (
+                rf"{pdf.label} ($68%$ c.l.+$1\sigma$)"
```
Comment (RoyStegeman) [inline validphys2/src/validphys/pdfplots.py:450]:
```suggestion
                rf"{pdf.label} ($68\%$ c.l.+$1\sigma$)"
```

File: validphys2/src/validphys/pdfplots.py [hunk 2]
```diff
@@ -438,11 +446,20 @@ def draw(self, pdf, grid, flstate):
...
+                if self.legend_stat_labels
+                else pdf.label
+            )
             outer = True
         else:
             outer = False
-            label = rf"{pdf.label} ($68\%$ c.l.)"
+            label = (
+                rf"{pdf.label} ($68\%$ c.l.)"
+                rf"{pdf.label} ($68%$ c.l.+$1\sigma$)"
```
Comment (RoyStegeman) [inline validphys2/src/validphys/pdfplots.py:459]:
```suggestion
```

Comment (Zaharid) [reply validphys2/src/validphys/pdfplots.py:459]: uh.

File: validphys2/src/validphys/pdfplots.py [hunk 3]
```diff
@@ -493,6 +510,9 @@ def plot_pdfs(
 
     show_mc_errors (bool): Plot 1σ bands in addition to 68% errors for Monte Carlo
     PDF.
+
+    legend_stat_labels (bool): Show detailed information on what kind of confidence
+    iterval is being plotted in the legend labels.
```
Comment (siranipour) [inline validphys2/src/validphys/pdfplots.py:515]:
```suggestion
    iNterval is being plotted in the legend labels.
```

**Comment by Zaharid** [comment]
Can this be merged?

File: validphys2/src/validphys/pdfplots.py [hunk 4]
```diff
@@ -547,7 +569,9 @@ def plot_lumi1d(
     exact bounds for the scale. `show_mc_errors` controls whether the 1σ error
     bands are shown in addition to the 68% confidence intervals for Monte Carlo
     PDFs. A list `pdfs_noband` can be passed to supress the error bands for
-    certain PDFs and plot the central values only.
+    certain PDFs and plot the central values only `legend_stat_labels` controls
```
Comment (RoyStegeman) [inline validphys2/src/validphys/pdfplots.py:572]:
```suggestion
    certain PDFs and plot the central values only. `legend_stat_labels` controls
```

**Review by RoyStegeman — APPROVED** [review]

**Review by siranipour — APPROVED** [review]

**Pull request merged**